### ISSUE TITLE
physlock input screen timeout

### ISSUE THREAD

**Issue by kartongsaft** [muennich/physlock#58]
physlock waits 60 seconds at the password prompt before sending a screen suspend/standby request. It would be nice if you had an option to control this, and even set this to 0, which immediately let the screen to sleep when physlock is activated.

**Comment by muennich** [comment]
Duplicate of issue #9?

**Comment by kartongsaft** [comment]
I'm not sure. The thing that I'm looking for, is to send screen standby request either as a function that determine environment and trigger the appropriate command or just trigger a second custom command/script.

**Comment by muennich** [comment]
I have no clue what you're talking about.

physlock has nothing to do with screen standby, as described in my comment on issue #9.

**Comment by kartongsaft** [comment]
Okay, let's take it from the begin. When I activated physlock earlier today, I got the black screen with a password prompt (as expected), then the screen stayed on for 60 seconds and then went in standby mode. I cannot explain why, but as I can't reproduce this, I think we just leave it there.

As for the custom command, I need to trigger custom command or script right after physlock has been activated. Could this be implemented?

**Comment by muennich** [comment]
Have a look at `-d` option. With it, `CMD` is run only after physlock set up everything:

    $ physlock -d; ${CMD}

I think this suffices for all kinds of setups, so that I do not see the need for a custom command mechanism in physlock itself.

**Comment by kartongsaft** [comment]
What I'm trying to achieve is to trigger a screen timeout using _xset dpms force standby_ or such, when physlock is activated. Is this even possible to do in when physlock is running?
